extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/OtlpRecorder.java

```diff
@@ -219,8 +219,12 @@ private void configureTLS(HttpClientOptions options) {
         }
 
         private KeyCertOptions toPemKeyCertOptions() {
-            PemKeyCertOptions pemKeyCertOptions = new PemKeyCertOptions();
             OtlpExporterTracesConfig.KeyCert keyCert = tracesConfig.keyCert();
+            if (keyCert.certs().isEmpty() && keyCert.keys().isEmpty()) {
```

Comment:
probably an OR, as it will likely not work properly if one of them is empty.

Reply:
It probably won't, but that is for Vert.x to sort out. 
In this case we are just trying to make Vert.x use it's defautl settings if the user has not configured anything, so we need `AND`